styles/hydrogen.less

```diff
@@ -360,8 +360,22 @@ svg:first-child {
 .hydrogen.inspector {
   padding: @component-padding;
   overflow-y: auto;
-
+  code {
+    font-family: var(--editor-font-family);
+  }
   span[style] {
     color: @text-color-info !important;
   }
 }
+
+.hydrogen.datatip {
```

Comment:
I want to keep them here since there are global `pre, code` style definition inside the `hydrogen` scope: https://github.com/nteract/hydrogen/blob/c046662301988657e91f3fd1a5a84cb340017e14/styles/hydrogen.less#L26

And overriding everything is boring for me.

Reply:
Ahh it works. Thanks !
I will refactor all of them.

Reply:
@wadethestealth 
But wait, can you explain why it _"should"_ be ?

Reply:
Less is a css preprocessor, and the whole reason they created the & was for this purpose. It creates cleaner stylesheets and can be more maintainable if names change around. For other benefits, read the docs. There is a bunch of neat feature less provides

Reply:
Thanks ! I got it :smile: